scripts/odemis-start-turret.py

```diff
@@ -64,16 +66,41 @@
 
 def main(args):
     turret = int(args[1])
+    notify = "--notify" in args[1:]
+    if len(args) >= 4:
+        name = args[3]
+    else:
+        name = f"turret {turret}"
     try:
+        if notify:
+            notify2.init("Odemis")
+
         pcu = powerctrl.PowerControlUnit(**KWARGS_PCU)
         logging.info("Turning on spectrograph... (2 min)")
-        pcu.supply({"Spectrograph": True}).result()
+        if notify:
+            notif = notify2.Notification("Starting Odemis", f"Selecting spectrograph {name}", icon="odemis")
+            notif.show()
+
+
+        # Trick: if the spectrograph is already on, let's assume it's already been on for a long time,
+        # so no need to turn it on again. This saves 90s... but if that's incorrect, the Shamrock
+        # driver might fail to initialize (but trying again will then work).
```

Comment:
Doesn't this refer to the option external/internal powered of the spectrograph? If so I think this comment could be rephrased.